index.html

```diff
@@ -90,4 +90,13 @@ <h3 class="post-subtitle">
      </article>
     {% endfor %}
   </div>
+
+  <!-- 2020 fundraiser sponsor logos -->
+  <div id="fundraiser">
+	  <h2 style="font-size:42px;">Supporters</h2>
+	  <p>Bitcoin ABC relies on sponsorships to fund all its work, including devops, maintenance, backports, development, release management, technical support and more. Become a supporter today at <a target=_fund" href="https://fund.bitcoinabc.org/">fund.bitcoinabc.org</a>.</p>
+	  <p><a target="_sponsor1" href="https://fund.bitcoinabc.org/"><img src="img/sponsors/sponsor.png" height="100" width="500" alt="Sponsor Bitcoin ABC today at fund.bitcoinabc.org" /></a></p>
+	  <p><a target ="_sponsor2" href="https://spicetoken.org/"><img src="img/sponsors/spice.png" height="200" width="175" alt="Visit spicetoken.org" /></a></p>
```

Comment:
The layout of the sponsors is a little wonky.  I highly recommend using flexbox, as it's industry standard now: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

See my patch, which gives you a good starting point using flexbox.